src/styles/Panel.css

```diff
@@ -13,18 +13,19 @@
 }
 
 .panel-collapse-button {
-  position: absolute;
-  top: 10px;
-  right: 10px;
   color: #dddcdf;
+  padding-right: 15px;
```

Comment:
add flex: 0 0 auto; to ensure that this doesn't get squished by the flex box when we run out of space
![image](https://user-images.githubusercontent.com/20308225/56014494-70e8a000-5caa-11e9-8452-6e9ab581d702.png)

Reply:
Added!